src/mscorlib/ref/mscorlib.cs

```diff
@@ -5730,7 +5730,7 @@ public partial class CultureInfo : System.ICloneable, System.IFormatProvider
         public static System.Globalization.CultureInfo GetCultureInfo(string name) { throw null; }
         public static System.Globalization.CultureInfo GetCultureInfo(string name, string altName) { throw null; }
         public static System.Globalization.CultureInfo GetCultureInfoByIetfLanguageTag(string name) { throw null; }
-        public static System.Globalization.CultureInfo[] GetCultures(System.Globalization.CultureTypes types) { throw null; }
```

Comment:
Sure, I'll let the builds finish first to make sure my CI is green, and then I'll add the comment and merge